src/components/ClassView/index.tsx

```diff
@@ -37,47 +38,54 @@ export class ClassView extends React.Component<IClassViewProps> {
       <div className="class-view">
         <h3>Plans: {classname}</h3>
         <p>Service Plans available for provisioning under {classname}</p>
-        <CardContainer>
-          {svcClass &&
-            classPlans.map(plan => {
-              const serviceClass = classes.find(
-                potential => potential.metadata.name === plan.spec.clusterServiceClassRef.name,
-              );
-              const { spec } = plan;
-              const { externalMetadata } = spec;
-              const name = externalMetadata ? externalMetadata.displayName : spec.externalName;
-              const description =
-                externalMetadata && externalMetadata.bullets
-                  ? externalMetadata.bullets
-                  : [spec.description];
-              const free = plan.spec.free ? <span>Free ✓</span> : undefined;
-              const bullets = (
-                <div>
-                  <ul>{description.map(bullet => <li key={bullet}>{bullet}</li>)}</ul>
-                </div>
-              );
+        <table>
+          <thead>
+            <tr>
+              <th>Name</th>
+              <th>Specs</th>
+              <th />
+            </tr>
+          </thead>
+          <tbody>
+            {svcClass &&
+              classPlans.map(plan => {
+                const serviceClass = classes.find(
+                  potential => potential.metadata.name === plan.spec.clusterServiceClassRef.name,
+                );
+                const { spec } = plan;
+                const { externalMetadata } = spec;
+                const name = externalMetadata ? externalMetadata.displayName : spec.externalName;
+                const description =
+                  externalMetadata && externalMetadata.bullets
+                    ? externalMetadata.bullets
+                    : [spec.description];
+                const bullets = (
+                  <div>
+                    <ul className="c-spec-list">
```

Comment:
you can use `margin-reset` or `remove-style. See https://bitnami-ui.now.sh/category/Foundations/Helpers/